blockchain/scriptval_test.go

```diff
@@ -4,37 +4,43 @@
 
 package blockchain
 
-// TODO(Litecoin): disable test until testfile is available
+import (
+	"fmt"
+	"testing"
+
+	"github.com/btcsuite/btcd/txscript"
+)
+
 // TestCheckBlockScripts ensures that validating the all of the scripts in a
 // known-good block doesn't return an error.
-// func TestCheckBlockScripts(t *testing.T) {
```

Comment:
re-enable tests! find testfiles!

Reply:
partially fixed. will work on for next release - requires utxostore file. need to figure out how to generate.